forcedotcom/kit
# Ensure support types are exported

[@W-12646003@](https://gus.my.salesforce.com/apex/ADM_WorkLocator?bugorworknumber=W-12646003)

Renames `support.d.ts` to `support.ts` so that the type declaration file is automatically added to `lib`. Removed the `postcompile` step

QA: 
- pull branch
- run `yarn install`
- run `yarn build`
- Ensure `lib/nodash/support.d.ts` exists

<img width="1093" alt="Screenshot 2023-03-06 at 12 30 45 PM" src="https://user-images.githubusercontent.com/1715111/223199360-2e7adf68-8183-4cc5-bb61-e809309ae026.png">
